### PR TITLE
Concepts

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -116,7 +116,10 @@ const sidebars = {
     "concepts/concepts-index",
     {
       "Service Overviews": [
+        "concepts/filter_service/filter-overview",
+        "concepts/gps_service/gps-overview",
         "concepts/controller_service/controller-overview",
+
```
Comment (Hackerman342) [inline website/sidebars.js:122]:
```suggestion
```

File: website/docs/concepts/filter_service/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+id: filter-overview
+title: Filter
+---
+
```
Comment (Hackerman342) [inline website/docs/concepts/filter_service/README.md:5]: Great overview -- I only think it is only missing information about requiring a connection to an RTK base station

File: website/docs/concepts/filter_service/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+(indicated by `has_converged` = true).
+
+:::info Remember
+State estimation is a dynamic process, and environmental factors or sensor issues can affect convergence.
+Regular calibration and adherence to the recommended workflows are essential for maintaining accurate
+state estimation.
+:::
+
+## API
+
```
Comment (Hackerman342) [inline website/docs/concepts/filter_service/README.md:53]: We should also link to `filter.proto` in farm-ng-amiga

File: website/sidebars.js [hunk 4]
```diff
@@ -116,7 +116,10 @@ const sidebars = {
     "concepts/concepts-index",
     {
       "Service Overviews": [
+        "concepts/filter_service/filter-overview",
+        "concepts/gps_service/gps-overview",
```
Comment (Hackerman342) [inline website/sidebars.js:120]: These should probably go roughly in complexity / dependency order
```suggestion
        "concepts/gps_service/gps-overview",
        "concepts/filter_service/filter-overview",
```

File: website/docs/concepts/gps_service/README.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+North, East, Down (N-E-D) frame. It's mainly used for applications requiring relative positioning
+between two receivers, often as a part of Real Time Kinematics (RTK) solutions.
+It shows the difference in position between a "moving" receiver and a "fixed" reference receiver.
+:::
+
+## Requirements
+
+For the GPS service to function optimally, certain conditions must be met:
+
+- The robot must have an active Wi-Fi connection, ensuring real-time data communication.
```
Comment (Hackerman342) [inline website/docs/concepts/gps_service/README.md:50]: Is a wifi connection required for PVT only operation? We should at least specify that pvt can be used without a base station

File: website/docs/concepts/gps_service/README.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+and systems to tap into accurate geospatial information without dealing with the
+complexities of GPS data interpretation.
+
+## Standalone Operation
+
+Unlike other services, the GPS service operates independently and is not a client of any other services.
+Its sole responsibility is to interpret and publish the data received from the GPS module.
+
+## Message Types
+
```
Comment (Hackerman342) [inline website/docs/concepts/gps_service/README.md:28]: link to `gps.proto` in farm-ng-amiga

**Review by Hackerman342 — APPROVED** [review]
Great overviews!

File: website/sidebars.js (hunk 4 above)
Comment (guilhermedemouraa) [reply website/sidebars.js:120]: I agree. I wasn't sure if we should organize it by alphabetical order or complexity.

File: website/docs/concepts/filter_service/README.md (hunk 2 above)
Comment (guilhermedemouraa) [reply website/docs/concepts/filter_service/README.md:5]: Addressed in commit da3ef391

File: website/docs/concepts/gps_service/README.md (hunk 6 above)
Comment (guilhermedemouraa) [reply website/docs/concepts/gps_service/README.md:28]: Addressed on commit 98971871

File: website/docs/concepts/gps_service/README.md (hunk 5 above)
Comment (guilhermedemouraa) [reply website/docs/concepts/gps_service/README.md:55]: Good point, I wasn't aware of this, but tested it on `boron-banana`.
Addressed this comment on commit bce87e5f

Comment (Hackerman342) [reply website/docs/concepts/gps_service/README.md:55]: Excellent! Thanks for testing

File: website/docs/concepts/gps_service/README.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+**RELPOSNED** (Relative Positioning) messages provide relative position information in a
+North, East, Down (N-E-D) frame. It's mainly used for applications requiring relative positioning
+between two receivers, often as a part of Real Time Kinematics (RTK) solutions.
+It shows the difference in position between a "moving" receiver and a "fixed" reference receiver.
+:::
+
+:::tip TIP
+For a comprehensive understanding of the list of variables available in each GPS message,
+make sure to check their [protobuff definitions](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/protos/farm_ng/gps/gps.proto).
```
Comment (mihelich) [inline website/docs/concepts/gps_service/README.md:48]:
```suggestion
make sure to check their [protobuf definitions](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/protos/farm_ng/gps/gps.proto).
```

File: website/docs/concepts/filter_service/README.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+The primary goal of the Filter service is to abstract the complexity involved in state estimation,
+providing users with clear, reliable information about the robot's pose, heading, and movement trajectory.
+This allows users to interact with and direct the Amiga effectively, without getting immersed in the
+technical depths of data fusion or sensor calibration.
+
+In essence, the Filter does the intricate work of synthesizing sensor data, enabling users to
+confidently interpret and utilize the Amiga's spatial information without needing extensive expertise
+in robotics or state estimation.
+
+## Subscritions
```
Comment (mihelich) [inline website/docs/concepts/filter_service/README.md:22]:
```suggestion
## Subscriptions
```

File: website/docs/concepts/filter_service/README.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+- Start with `/clear_track`. This will clear the current track on memory and allow the user to
+"start fresh"
+- User drives the robot to generate the desired track
+- Once satisfied, the user stops the robot
+- Get the track with the API `/get_track`.
+
+:::tip TIP
+For a comprehensive understanding of the list of variables available in the filter state messages,
+make sure to check its [protobuff definition](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/protos/farm_ng/filter/filter.proto).
```
Comment (mihelich) [inline website/docs/concepts/filter_service/README.md:78]:
```suggestion
make sure to check its [protobuf definition](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/protos/farm_ng/filter/filter.proto).
```

**Review by mihelich — APPROVED** [review]
These are super helpful, thanks @guilhermedemouraa! I learned some things.

Just a few typo suggestions. We'll eventually want more info on setting up a base station, but this is great for now.

**Comment by mihelich** [comment]
Oak overview looks good. I'm going to go ahead and merge the content so far. We can do follow-up PRs for additional content like CANBUS.

**Pull request merged**